src/udp_wrap.cc

```diff
@@ -213,6 +214,25 @@ X(SetMulticastLoopback, uv_udp_set_multicast_loop)
 
 #undef X
 
+void UDPWrap::SetMulticastInterface(const FunctionCallbackInfo<Value>& args) {
+  UDPWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap,
+                          args.Holder(),
+                          args.GetReturnValue().Set(UV_EBADF));
+
+  CHECK_EQ(args.Length(), 1);
+
+  node::Utf8Value iface(args.GetIsolate(), args[0]);
+
+  const char* iface_cstr = *iface;
+  if (args[0]->IsUndefined() || args[0]->IsNull()) {
```

Comment:
`args[0]->IsUndefined()` seems to be untouchable because we checked the address at the [JS level](https://github.com/nodejs/node/pull/7855/files#diff-5c0ca4f44209cc4cc68c4dccadbd4a07R501).